config/clients/python/template/partial_header.mustache

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
Can you use content similar to the one in common:
https://github.com/openfga/sdk-generator/blob/main/config/common/files/partial_header.mustache

Reply:
Done